src/sources/forus-platform/pug/tpl/printables/printable-voucher-qr_code.pug

```diff
@@ -1,10 +1,40 @@
 //- Printable class is required 
 .printable.printable-voucher-qr_code
-    .printable-title(ng-if="$ctrl.voucher.type != 'product'") Uw {{ $ctrl.voucher.fund.name }}-voucher
-    .printable-title(ng-if="$ctrl.voucher.type == 'product'") Uw {{ $ctrl.voucher.product.name }}-voucher
-    br
-    qr-code(qr-value="$ctrl.qrCodeValue" qr-type="voucher")
-    .printable-description De verloopdatum van deze voucher is {{ $ctrl.voucher.expire_at_locale }}
+    .printable-main-info
+        .printable-usage-info 
+            span(i18n="voucher_printable.{{$ctrl.voucherType}}_voucher.labels.use_for")
+            br
+            strong {{ $ctrl.voucher.expire_at_locale }}
```

Comment:
expire_at is missing font-family.
![image](https://user-images.githubusercontent.com/10818702/69963397-6ff1d080-1510-11ea-905a-5691de2ae727.png)